src/main/scala/mesosphere/marathon/core/event/impl/stream/HttpEventStreamServlet.scala

```diff
@@ -32,6 +34,26 @@ class HttpEventSSEHandle(request: HttpServletRequest, emitter: Emitter, allowHea
   else
     true
 
+  /**
+    * Calculates the number of bytes sent (including the SSE framing) and updates
+    * the `bytesWrittenMetric` accordingly.
+    *
+    * Note that an SSE frame has the following structure:
+    *
+    * "name: <name>\n
+    *  data: <payload>\n
+    *  \n
+    * "
+    *
+    * @param eventName The name of the event, used for estimating the "name" field length
+    * @param payload The event payload, used for estimating the "data" field length
+    * @return Passes through the `payload` argument
+    */
+  private def measureFrameBytesSent(eventName: String, payload: String): String = {
```

Comment:
Nit: I don't think the return type is correct.

Reply:
yup, fixed it, forgot to push 😄